### PR TITLE
Closes#3185 Migrate strings xml entities to CDATA

### PR DESCRIPTION
**Pull request by keianhzo**
Closes#3185 Migrate strings xml entities to CDATA

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -632,7 +632,9 @@
 
         <!-- This string labels an Allow/Don't Allow switch in the Privacy and Security settings dialog
      and is used to enable or disable playback of DRM controlled content. -->
-    <string name="security_options_drm_content_v1">Automatically allow sites to identify your device to play DRM-controlled content. (&lt;a href="%1$s">Learn More&lt;/a>)</string>
+    <string name="security_options_drm_content_v1"><![CDATA[
+Automatically allow sites to identify your device to play DRM-controlled content. (<a href="%1$s">Learn More</a>)
+     ]]></string>
```
Comment (bluemarvin) [inline app/src/main/res/values/strings.xml:637]: Do the `name` versions need to be bumped for this change?

Comment (Pike) [inline app/src/main/res/values/strings.xml:637]:
```suggestion
    <string name="security_options_drm_content_v1"><![CDATA[Automatically allow sites to identify your device to play DRM-controlled content. (<a href="%1$s">Learn More</a>)]]></string>
```

Comment (keianhzo) [reply app/src/main/res/values/strings.xml]: @Pike updated

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1653,7 +1653,7 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <!-- This string is displayed in the title of the quick DRM dialog, accessed from the DRM icon in the URL bar.
          '%1$s' will be replaced at runtime with the status of the setting.
          '%2$s' will be replaced at runtime with the Learn More page link. -->
-    <string name="drm_dialog_message">Some audio or video on this site uses DRM software, which may limit what %1$s can let you do with it. &lt;a href="%2$s">Learn More&lt;/a></string>
+    <string name="drm_dialog_message"><![CDATA[Some audio or video on this site uses DRM software, which may limit what %1$s can let you do with it. <a href="%1$s">Learn More</a>]]></string>
```
Comment (Pike) [inline app/src/main/res/values/strings.xml:1656]: `href="%2$s"`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1642,7 +1642,7 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <!-- This string is displayed in the title of the quick Tracking Protection dialog, accessed from the Tracking Protection icon in the URL bar.
          '%1$s' will be replaced at runtime with the status of the setting.
          '%2$s' will be replaced at runtime with the Learn More page link. -->
-    <string name="tracking_dialog_message">Enhanced Tracking Protection is %1$s for this site. &lt;a href="%2$s">Learn More&lt;/a></string>
+    <string name="tracking_dialog_message"><![CDATA[Enhanced Tracking Protection is %1$s for this site. <a href="%1$s">Learn More</a>]]></string>
```
Comment (Pike) [inline app/src/main/res/values/strings.xml:1645]: `href="%2$s"`

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1442,7 +1442,7 @@
 
     <!-- This string is displayed in the description of the Voice Search samples collect dialog. That dialog is shown the first time
          a user tries to use the voice search. -->
-    <string name="voice_samples_collect_dialog_description2">In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection.&lt;a href="privacy">Learn More&lt;/a></string>
+    <string name="voice_samples_collect_dialog_description2"><![CDATA[In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection. <a href="%1$s">Learn More</a>]]></string>
```
Comment (Pike) [inline app/src/main/res/values/strings.xml:1445]: `href="privacy"`

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (keianhzo) [reply app/src/main/res/values/strings.xml]: Fixed

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (keianhzo) [reply app/src/main/res/values/strings.xml]: We don't really use the variable there, I've removed it.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (keianhzo) [reply app/src/main/res/values/strings.xml]: Fixed

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1442,7 +1444,7 @@
 
     <!-- This string is displayed in the description of the Voice Search samples collect dialog. That dialog is shown the first time
          a user tries to use the voice search. -->
-    <string name="voice_samples_collect_dialog_description2">In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection.&lt;a href="privacy">Learn More&lt;/a></string>
+    <string name="voice_samples_collect_dialog_description2"><![CDATA[In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection. <a href="">Learn More</a>]]></string>
```
Comment (Pike) [inline app/src/main/res/values/strings.xml:1447]:
```suggestion
    <string name="voice_samples_collect_dialog_description2"><![CDATA[In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection. <a href="privacy">Learn More</a>]]></string>
```

Comment (Pike) [reply app/src/main/res/values/strings.xml:1447]: The added ` ` is OK w/out ID change, but the link got lost in this one still.

Comment (keianhzo) [reply app/src/main/res/values/strings.xml:1447]: That one doesn't require a link, it's handled internally, so it should be good.

**Comment by keianhzo** [comment]
@Pike Is this ready to be merged?

**Pull request merged**